openfisca_us/variables/gov/states/in/tax/income/exemptions/in_aged_blind_count.py

```diff
@@ -0,0 +1,20 @@
+from openfisca_us.model_api import *
+
+
+class in_aged_blind_count(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "IN aged and or blind head and spouse count"
+    unit = USD
+    definition_period = YEAR
+    reference = "http://iga.in.gov/legislative/laws/2021/ic/titles/006#6-3"
```

Comment:
Could you add a comment with the specific subsection? Looks like `6-3-1-3.5(a)(4)(B)` in this case.